plugins/axiom/personal_access_token.go

```diff
@@ -0,0 +1,40 @@
+package axiom
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func PersonalAccessToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.PersonalAccessToken,
+		DocsURL:       sdk.URL("https://axiom.co/docs/restapi/token#creating-personal-token"),
+		ManagementURL: sdk.URL("https://app.axiom.co/settings/profile"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Token,
+				MarkdownDescription: "Token used to authenticate to Axiom.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 41,
+					Prefix: "xapt-",
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.EnvVars(defaultEnvVarMapping),
+		Importer: importer.TryAll(
+			importer.TryEnvVarPair(defaultEnvVarMapping),
+		)}
```

Comment:
Let's use `importer.TryEnvVarPair(defaultEnvVarMapping)` directly, no need for `TryAll` for only one importer. 😄